cluster-operator/src/test/java/io/strimzi/operator/cluster/model/ModelUtilsTest.java

```diff
@@ -302,61 +312,166 @@ public void testValidHeapPercentage() {
         assertThat(env.get(AbstractModel.ENV_VAR_KAFKA_HEAP_OPTS), is("-Xms4"));
         assertThat(env.get(AbstractModel.ENV_VAR_DYNAMIC_HEAP_PERCENTAGE), is(nullValue()));
         assertThat(env.get(AbstractModel.ENV_VAR_DYNAMIC_HEAP_MAX), is(nullValue()));
     }
 
     @ParallelTest
-    public void testJvmMemoryOptionsXmxOnly() {
+    void testJvmMemoryOptionsXmxOnly() {
         Map<String, String> env = heapOptions(jvmOptions("4", null), 50, 5_000_000_000L, null);
         assertThat(env.get(AbstractModel.ENV_VAR_KAFKA_HEAP_OPTS), is("-Xmx4"));
         assertThat(env.get(AbstractModel.ENV_VAR_DYNAMIC_HEAP_PERCENTAGE), is(nullValue()));
         assertThat(env.get(AbstractModel.ENV_VAR_DYNAMIC_HEAP_MAX), is(nullValue()));
     }
 
     @ParallelTest
-    public void testJvmMemoryOptionsDefaultWithNoMemoryLimitOrJvmOptions() {
+    void testJvmMemoryOptionsDefaultWithNoMemoryLimitOrJvmOptions() {
         Map<String, String> env = heapOptions(jvmOptions(null, null), 50, 5_000_000_000L, null);
         assertThat(env.get(AbstractModel.ENV_VAR_KAFKA_HEAP_OPTS), is("-Xms" + ModelUtils.DEFAULT_JVM_XMS));
         assertThat(env.get(AbstractModel.ENV_VAR_DYNAMIC_HEAP_PERCENTAGE), is(nullValue()));
         assertThat(env.get(AbstractModel.ENV_VAR_DYNAMIC_HEAP_MAX), is(nullValue()));
     }
 
     @ParallelTest
-    public void testJvmMemoryOptionsDefaultWithMemoryLimit() {
+    void testJvmMemoryOptionsDefaultWithMemoryLimit() {
         Map<String, String> env = heapOptions(jvmOptions(null, "4"), 50, 5_000_000_000L, new ResourceRequirementsBuilder().withLimits(Map.of("memory", new Quantity("1Gi"))).build());
         assertThat(env.get(AbstractModel.ENV_VAR_KAFKA_HEAP_OPTS), is("-Xms4"));
         assertThat(env.get(AbstractModel.ENV_VAR_DYNAMIC_HEAP_PERCENTAGE), is("50"));
         assertThat(env.get(AbstractModel.ENV_VAR_DYNAMIC_HEAP_MAX), is("5000000000"));
     }
 
     @ParallelTest
-    public void testJvmMemoryOptionsMemoryRequest() {
+    void testJvmMemoryOptionsMemoryRequest() {
         Map<String, String> env = heapOptions(null, 70, 10_000_000_000L, new ResourceRequirementsBuilder().withRequests(Map.of("memory", new Quantity("1Gi"))).build());
         assertThat(env.get(AbstractModel.ENV_VAR_KAFKA_HEAP_OPTS), is(nullValue()));
         assertThat(env.get(AbstractModel.ENV_VAR_DYNAMIC_HEAP_PERCENTAGE), is("70"));
         assertThat(env.get(AbstractModel.ENV_VAR_DYNAMIC_HEAP_MAX), is("10000000000"));
     }
 
     @ParallelTest
-    public void testServiceDnsNames() {
+    void testServiceDnsNames() {
         List<String> dnsNames = ModelUtils.generateAllServiceDnsNames("my-namespace", "my-service");
 
         assertThat(dnsNames.size(), is(4));
         assertThat(dnsNames, hasItems("my-service", "my-service.my-namespace", "my-service.my-namespace.svc", "my-service.my-namespace.svc.cluster.local"));
     }
 
+    @ParallelParameterizedTest
```

Comment:
Does all this parametrization really add something super special what is worth it? Would it be more useful to just create a private helper method and pass the parameters into it one by one? For Strimzi as a project is not really useful to have 3 unit tests that work in some completely special way. It just makes it harder to maintain the project and do the next contributions.

Reply:
I think parameterized tests can be used in other places, I generally find them quite useful, but I can remove it.

Reply:
Removed